src/Components/HighlightText.re

```diff
@@ -5,6 +5,8 @@ let make =
       ~text: string,
       ~highlights: list((int, int)),
       ~style,
+      ~fontFamily=?,
+      ~fontSize=?,
       ~highlightStyle,
       (),
     ) => {
```

Comment:
There's a case here that's missed - for the last element - we need to also set `?fontFamily` and `?fontSize` on this text node, too